BEIMA.Backend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS installer-env
+EXPOSE 7071
+# Build requires 3.1 SDK
+COPY --from=mcr.microsoft.com/dotnet/core/sdk:3.1 /usr/share/dotnet /usr/share/dotnet
```

Comment:
Why do we need 3.1?

Reply:
should it be different? this is the 6th or 7th iteration of tryin to get this one to work, this was the config that finally did

Reply:
I believe we have been using .net 6.0 if I'm not mistaken

Reply:
It's the SDK/Framework version vs the C# version.
C# is at version 6, .NET Core is the cross platform version of the .NET framework whereas the standard .NET Framework is Windows only.

As far as I'm aware, without .NET Core our app won't run.

Reply:
I believe that what is now just "dotnet" used to be "dotnet core". You are correct in that it used to be the case that dotnet core was used for multiplatform environments and that framework was used for Windows. However, my understanding was that after dotnet core 3.1 and some framework version, they stopped splitting releases into dotnet core and framework and just made one "dotnet".

Reply:
Does it not work if you don't include dotnet Core 3.1?

Reply:
Just tried it without the copy and in the MS provided container it does!
I was thinking it would still be required since in one of the iterations (1 -> 5 or so) when I was building it on top of the ubuntu 20.04 image manually/directly it complained about the dotnet command not being there until the SDK was added too.

Looks like in the MS provided container there's enough already installed to do it without it.

Reply:
build for the API was completely changed, but SDK 6 was required